src/libertem/web/cli.py

```diff
@@ -51,6 +53,11 @@ def get_token(token_path):
               default=False, is_flag=True)
 def main(port, local_directory, browser, cpus, gpus, open_ds, log_level,
          insecure, host="localhost", token_path=None, preload: Tuple[str, ...] = ()):
+    # Mitigation for https://stackoverflow.com/questions/71283820/
+    #   directory-parameter-on-windows-has-trailing-backslash-replaced-with-double-quote
+    if (open_ds and platform.system() == 'Windows' and open_ds[-1] == '"'
```

Comment:
:sob:

Reply:
wInDoWs iS gReAt!1! :-D

Reply:
Coincidentally ran into practically the same bug today in a completely different context (DM launching a python command) - the final `\'` in the Python command was interpreted as an escaped quotation mark leading to `SyntaxError`.

Reply:
Yup, that's also a nice edge case of raw strings - `r'c:\stuff\meh'` works but `r'c:\stuff\meh\'` needs an additional backslash for escaping, `r'c:\stuff\meh\\'`